src/components/node/sul-library/page-display.tsx

```diff
@@ -8,31 +8,30 @@ const SulLibrary = async ({node, ...props}: { node: Library }) => {
   node.su_library__paragraphs = await fetchComponents<StanfordParagraph>(node.su_library__paragraphs || []);
   node.su_library__paragraphs = node.su_library__paragraphs.filter(item => !!item?.id);
   const fullWidth = node.layout_selection?.resourceIdObjMeta?.drupal_internal__target_id === 'sul_library_full_width'
+  const containerClasses = ["mb-50", "lg:max-w-[980px]", "mx-auto", "centered"];
+  const containerStyles = containerClasses.join(' ');
```

Comment:
Just put these right in the elements. Even if it is duplicated. Creating a variable like this just adds confusion.

Reply:
https://www.npmjs.com/package/cnbuilder is commonly used by our other FE devs.

Reply:
Now that I'm digging in cnbuilder, I suddenly feel under-qualified to make an assessment...

Reply:
You should follow the convention of the project you are in. cnbuilder can help with the concatenation and duplication of styles within a set.

Reply:
> You should follow the convention of the project you are in

Sure, but shouldn't you also be able to provide improvements. (I'm just not qualified to.)

Reply:
At least not yet. I hope someday soon to be able to make a contribution in that regards.

Reply:
> Sure, but shouldn't you also be able to provide improvements. (I'm just not qualified to.)

Yeah, we should be open to changes or differences but this thread isn't the place to have that. The impact of this style approach would be too large and it would block the work at hand. I'd suggest you open up a new ticket to explore more and start a conversation. Ask questions and see what others are doing.

Reply:
> I'd suggest you open up a new ticket to explore more and start a conversation.

That seems like a fair approach.